packages/foam-core/test/markdown-provider.test.ts

```diff
@@ -148,6 +149,42 @@ this is a [link to intro](#introduction)
     expect(link.label).toEqual('spaced');
     expect(link.target).toEqual('other link');
   });
+
+  it('Skips wikilinks in codeblocks', () => {
```

Comment:
somewhat unrelated to the PR, but given I added tests for tags in codeblocks, I took the chance to add these.
No changes to wikilinks have been made